ModPatches/Rimsenal Feral/Patches/Rimsenal Feral/ThingDefs_Misc/Feral_MutantWeapons.xml

```diff
@@ -118,15 +118,15 @@
 	<Operation Class="CombatExtended.PatchOperationMakeGunCECompatible">
 		<defName>Feral_RivetCannon</defName>
 		<statBases>
-			<Mass>5.3</Mass>
+			<Mass>8</Mass>
 			<RangedWeapon_Cooldown>0.38</RangedWeapon_Cooldown>
 			<SightsEfficiency>0.89</SightsEfficiency>
 			<ShotSpread>0.17</ShotSpread>
-			<SwayFactor>1.7</SwayFactor>
+			<SwayFactor>1.26</SwayFactor>
 			<Bulk>11.03</Bulk>
 		</statBases>
 		<Properties>
-			<recoilAmount>1.83</recoilAmount>
+			<recoilAmount>0.92</recoilAmount>
 			<verbClass>CombatExtended.Verb_ShootCE</verbClass>
 			<hasStandardCommand>true</hasStandardCommand>
 			<defaultProjectile>Bullet_Feral_Nail</defaultProjectile>
```

Comment:
Needs to be switched to the new Rivet projectile.

Reply:
Fixed